### ISSUE TITLE
unsupported number of color planes

### ISSUE THREAD

**Issue by alexmyczko** [pete-gordon/hivelytracker#24]
when running the sdl2 version on linux, i'm getting this error message
Skins/Vintage/prefs_cycle.png': unsupported number of color planes

when changing the theme to Vintage

$ file */*cycle.png
SIDMonster-Light/prefs_cycle.png: PNG image data, 160 x 48, 8-bit colormap, non-interlaced
Vintage/prefs_cycle.png:          IFF data, ILBM interleaved image, 160 x 48
$ feh -l */*cycle.png
NUM	FORMAT	WIDTH	HEIGHT	PIXELS	SIZE	ALPHA	FILENAME
1	png	160	48	  7k	  1k	-	SIDMonster-Light/prefs_cycle.png
2	lbm	160	48	  7k	578 	X	Vintage/prefs_cycle.png

**Comment by alexmyczko** [comment]
find my tested+working fix at: https://github.com/alexmyczko/hivelytracker/commit/5a91c9bab33c13a6de12b36b37fc95d4fd087978

**Comment by alexmyczko** [comment]
I will try

**Comment by pete-gordon** [comment]
Seems like the pull request only deletes "prefs_cycle.png". There is no rename, or inclusion of a new png file.

**Comment by alexmyczko** [comment]
Ah answered at the wrong place. Probably easier if you just add mine from my fork?

**Comment by pete-gordon** [comment]
Ok but I don't know when I'll have time... I can merge pull requests from my phone.

**Comment by alexmyczko** [comment]
I can't figure out how to use the github web interface to replace a binary file. I am not allowed to delete it first, to add the good one. Can you delete files from your phone?

**Comment by pete-gordon** [comment]
No. I would prepare the fix on my laptop using the got commandline. Hence the need for me to have some time to actually do it.

**Comment by alexmyczko** [comment]
Sounds good, I'm not in a hurry. I do have it in the debian package already, and can get rid of the patch with your new version later any time: https://packages.debian.org/hivelytracker

**Comment by alexmyczko** [comment]
meanwhile i can use github without phone or browser. achivement git cli guru. pr following
